src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -336,19 +337,13 @@ public void verifyVideoAlignment(PositionsVideo position) {
     );
   }
 
-  public void verifyVideoWidth(int widthDesired) {
+  public int getVideoWidth() {
     String content = getContent();
     int width = Integer.parseInt(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "width". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1488)